plugin/src/geolocation.ts

```diff
@@ -0,0 +1,52 @@
+import { PostMessageProps } from "@web/extensions/geolocation/core/types";
+
+import html from "../dist/web/geolocation/core/index.html?raw";
+
+const reearth = (globalThis as any).reearth;
+
+reearth.ui.show(html, { width: 44, height: 44 });
+
+reearth.on("message", ({ action, payload }: PostMessageProps) => {
+  if (action === "flyTo") {
+    if (
+      payload.currentLocation.latitude !== undefined &&
+      payload.currentLocation.longitude !== undefined &&
+      payload.currentLocation.altitude !== undefined
+    ) {
```

Comment:
Since there are no `else` or `else/if` statements just combine these if statements.
But also, if currentLocation.latitude/longitube/altitude can be undefined, please update the `CurrentLocationInfo` type to be more accurate. If they will always have a value, you don't need to check if they are undefined.

Reply:
done

Reply:
fc8de4437577d111f90b0a1c7db09d37c48d4de6